bin/install_openstackbmc.sh

```diff
@@ -45,12 +45,25 @@ export OS_TENANT_NAME=$os_tenant
 export OS_PASSWORD=$os_password
 export OS_AUTH_URL=$os_auth_url
 export OS_PROJECT_NAME=$os_project
-export OS_USER_DOMAIN_ID=$os__user_domain
-export OS_PROJECT_DOMAIN_ID=$os__project_domain
+export OS_USER_DOMAIN_ID=$os_user_domain
```

Comment:
@cybertron can't we use ${variable} instead of $variable($variable2) use case that you poited out. I think it would be more clear to use this convention and use os_user_domain instead of os__user_domain. What do you think?

<<< I did just notice that I missed changing this in the systemd service file. Is that the problem you ran into as well?

Yes the service didn't got started without correct project_domain and user_domain.